libs/core/src/lib/label/label-sets/label-sets.component.ts

```diff
@@ -28,11 +31,23 @@ export class LabelSetsComponent {
   selectionLabelSets: Observable<LabelSetDisplay[]> = this._labelSets.pipe(
     map((labelSets) => labelSets.filter((labelSet) => labelSet.kind.includes(LabelSetKind.SELECTIONS))),
   );
+  showingLabelSetForm = this.router.events.pipe(
+    filter((event) => event instanceof NavigationEnd),
+    startWith(() => true),
+    map(() => this.route.children.length > 0),
+  );
```

Comment:
it's not obvious and a bit hard to understand the causality between the route’s children and the fact we show label set form. Is there any simpler approach?

Reply:
oh, yes, it's not very obvious.
The only idea that comes to mind is to use `location.pathname.endsWith('/label-sets') === false`.
would it be clearer ?

Reply:
could you achieve the same using `activatedRoute` instead of location? It's always better to stick with Angular routing component than browser location.

Reply:
ok, now it's fixed